packages/cli/src/lib/print-config/print-config-command.unit.test.ts

```diff
@@ -31,30 +27,13 @@ describe('print-config-command', () => {
       { ...DEFAULT_CLI_CONFIGURATION, commands: [yargsConfigCommandObject()] },
     ).parseAsync();
 
-    const log = ui().logger.getLogs()[0];
-
-    expect(log).toEqual(
-      expect.objectContaining({
-        message: expect.not.stringContaining('"$0":'),
-      }),
-    );
-
-    expect(log).toEqual(
-      expect.objectContaining({
-        message: expect.not.stringContaining('"_":'),
-      }),
-    );
-
-    expect(log).toEqual(
-      expect.objectContaining({
-        message: expect.stringContaining('"outputDir": "destinationDir"'),
-      }),
-    );
+    const log = getLogMessages(ui().logger)[0];
+    expect(log).not.toContain('"$0":');
+    expect(log).not.toContain('"_":');
 
     expect(log).toEqual(
-      expect.objectContaining({
-        message: expect.not.stringContaining('"output-dir":'),
-      }),
+      expect.stringContaining('"outputDir": "destinationDir"'),
     );
+    expect(log).toEqual(expect.not.stringContaining('"output-dir":'));
```

Comment:
When "applicable in other test cases" is written in a review comment, please go through other tests and check where else that pattern can be used.

In this case, please apply `toContain` and `not.toContain` as discussed. Please look at other files where this is applicable (this is not the only place).

Reply:
can't this be a lint rule?

Adopted ✅

Reply:
I am sure that technically it could be. Not sure if it exists, feel free to research.

Related comment: https://github.com/code-pushup/cli/pull/548#issuecomment-1985379373